yarn-project/sequencer-client/src/simulator/rollup.ts

```diff
@@ -33,22 +33,22 @@ export class WasmRollupCircuitSimulator implements RollupSimulator {
    * @returns The public inputs as outputs of the simulation.
    */
   baseRollupCircuit(input: BaseRollupInputs): Promise<BaseOrMergeRollupPublicInputs> {
-    return this.rollupWasmWrapper.simulateBaseRollup(input);
```

Comment:
Some stuff we want the flexibility of async in the future, Promise.resolve is an idiom for that